user-console/src/main/java/org/pentaho/mantle/client/admin/ContentCleanerPanel.java

```diff
@@ -236,6 +236,7 @@ public void cancelPressed() {
               editSchedule.setShowSuccessDialog( false );
               editSchedule.addCustomPanel( scheduleLabelPanel, DockPanel.NORTH );
               editSchedule.center();
+              scheduleTextBox.setFocus( true );
```

Comment:
AFAICT, `scheduleTextBox` is the first "tabbable" element in the dialog's content. Is it not being focused by the default focus handling? Why not? Is it a bug of the default focus code?

If it is not the first element, then the expected way to change the default focus widget is to call the `editSchedule.setFocusWidget( scheduleTextBox )` before the center method.